tests/test_controller.py

```diff
@@ -552,6 +560,10 @@ def test_sitewide_key_pair(self):
         # Calling it again will do nothing.
         eq_((new_public, new_private), self.manager.sitewide_key_pair)
 
+    def test_annotator(self):
+        # TODO.
+        pass
```

Comment:
Did you mean to leave this?

Reply:
Yeah, I noticed there was no coverage for this method but originally intended to leave it out. However since I'm changing this method (to move the CirculationManagerAnnotator code into it) I will add the coverage.